Xamarin.Forms.Platform.UAP/Shell/ShellSectionRenderer.cs

```diff
@@ -191,7 +175,49 @@ internal void NavigateToContent(ShellNavigationSource source, ShellContent shell
 			}
 		}
 
-		NavigationTransitionInfo GetTransitionInfo(ShellNavigationSource navSource)
+		protected virtual void OnPopRequested(ShellNavigationSource source)
```

Comment:
Would these be more helpful if they included the full arg like iOS?

https://github.com/xamarin/Xamarin.Forms/blob/a695fabafa51cda581c9d2f24c9a4f68ddc6bb14/Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRenderer.cs#L328

Reply:
yepp, is a good idea.
Do you think we should use the `NavigationRequestedEventArgs` in Xamarin.Forms.Core? Or just pass the Page and isAnimated properties in the virtual method?

I'm asking because the `NavigationRequestedEventArgs` has this property:
`public Task<bool> Task { get; set; }`
but in the shell renderer we dont have a navigation task

Reply:
I think NavigationRequestedEventArgs makes sense

We use that property on the iOS renderers, it might be useful on UWP at some point